Packs/Ansible_Powered_Integrations/Integrations/AzureComputeV3/AzureComputeV3.yml

```diff
@@ -943,130 +1002,132 @@ script:
         disk to copy.
       name: source_uri
     - auto: PREDEFINED
-      description: |-
-        Type of Operating System.
-        Used when `create_option=copy` or `create_option=import` and the source is an OS disk.
+      description: 'Type of Operating System.
+
+        Used when `create_option=copy` or `create_option=import` and the source is
+        an OS disk.
+
         If omitted during creation, no value is set.
+
         If omitted during an update, no change is made.
-        Once set, this value cannot be cleared.
+
+        Once set, this value cannot be cleared.'
       name: os_type
       predefined:
       - linux
       - windows
-    - description: |-
-        Size in GB of the managed disk to be created.
-        If `create_option=copy` then the value must be greater than or equal to the source's size.
+    - description: 'Size in GB of the managed disk to be created.
+
+        If `create_option=copy` then the value must be greater than or equal to the
+        source''s size.'
       name: disk_size_gb
-    - description: |-
-        Name of an existing virtual machine with which the disk is or will be associated, this VM should be in the same resource group.
-        To detach a disk from a vm, explicitly set to ''.
-        If this option is unset, the value will not be changed.
+    - description: 'Name of an existing virtual machine with which the disk is or
+        will be associated, this VM should be in the same resource group.
+
+        To detach a disk from a vm, explicitly set to ''''.
+
+        If this option is unset, the value will not be changed.'
       name: managed_by
     - auto: PREDEFINED
-      description: |-
-        Disk caching policy controlled by VM. Will be used when attached to the VM defined by `managed_by`.
-        If this option is different from the current caching policy, the managed disk will be deattached and attached with current caching option again.
+      description: 'Disk caching policy controlled by VM. Will be used when attached
+        to the VM defined by `managed_by`.
+
+        If this option is different from the current caching policy, the managed disk
+        will be deattached and attached with current caching option again.'
       name: attach_caching
       predefined:
-      - ""
+      - ''
```

Comment:
The [source Ansible module](https://docs.ansible.com/ansible/2.9/modules/azure_rm_manageddisk_module.html#parameter-attach_caching) lists ''' as an option....  Seems like a typo to me. I'm tempted to fix it upstream rather than maintain a list of manually applied changes. Lets discuss further when we next catchup